src/Helpers/CVEHelper.js

```diff
@@ -73,7 +73,7 @@ export function createCveDetailsPage(cves) {
 export function createRHDBLink(item) {
     return (
         <a target="_blank" rel="noopener noreferrer" href={'https://access.redhat.com/security/cve/' + item}>
-            {<FormattedMessage {...messages.RHDBLink} />}{<ExternalLinkAltIcon />}
+            {<FormattedMessage {...messages.RHDBLink} />}{' '}{<ExternalLinkAltIcon />}
```

Comment:
I suggest instead of using the empty string `{' '}`  between the label and the icon to use 
something like `style="margin-left: var(--pf-global--spacer--sm);"`  for the icon 
or create a general class the can be used across the app for such cases